aiida_vasp/utils/tests/test_aiida_utils.py

```diff
@@ -10,10 +10,11 @@
 
 
 def test_get_current_user(fresh_aiida_env):
-    """Assert that get_current_user returns a user in all tested aiida versions."""
+    """Assert that get_current_user returns a user."""
```

Comment:
What is the aim of this test?

Reply:
@atztogo I think this is back from when AiiDA had limited test fixtures available for the plugins. I assume, now, that the test fixtures are tested in AiiDA, we can slim this down. I think this goes for all the tests in `test_aiida_utils.py`. Also, checking that the entrypoints are okey is also less of a concern now due to the removal of `reentry scan`. If we agree, I will simply remove this file altogether.